### PR TITLE
Complete cluster config removal from the auth service

### PR DESCRIPTION
**Pull request by giorio94**
# Description

This PR completes the removal of the cluster config parameters from the authentication service, and refactors its flags to align them to the format used by the other components.

Fixes #(issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [x] Unit testing (existing)
- [x] E2E testing (existing)

### CODE REVIEW

**Review by palexster — APPROVED** [review]

File: deployments/liqo/templates/liqo-auth-deployment.yaml [hunk 1]
```diff
@@ -70,25 +70,31 @@ spec:
...
+          - --address=:5000
+          {{- else }}
+          - --address=:8443
+          - --enable-tls
+          {{- end }}
+          - --enable-authentication={{ .Values.auth.config.enableAuthentication }}
+          {{- if .Values.apiServer.address }}
+          - --advertise-api-server-address={{ .Values.apiServer.address }}
+          - --advertise-api-server-trusted-ca={{ .Values.apiServer.trustedCA }}
+          {{- end }}
```
Comment (aleoli) [inline deployments/liqo/templates/liqo-auth-deployment.yaml:83]:
```suggestion
          {{- if .Values.apiServer.address }}
          - --advertise-api-server-address={{ .Values.apiServer.address }}
          {{- end }}
          {{- if .Values.apiServer.trustedCA }}
          - --advertise-api-server-trusted-ca={{ .Values.apiServer.trustedCA }}
          {{- end }}
```

It's probably clearer and more flexible

**Review by aleoli — APPROVED** [review]

**Pull request merged**